src/arviz_stats/visualization.py

```diff
@@ -370,13 +373,14 @@ def histogram(
     Returns
     -------
     ndarray, DataArray, Dataset, DataTree
-        Requested HDI of the provided input.
+        Requested histogram of the provided input.
         It will have a ``{var_name}_hist_dim`` dimension and a ``plot_axis`` dimension
```

Comment:
reading this I realized [I switched the order of prefix/name](https://github.com/arviz-devs/arviz-stats/blob/main/src/arviz_stats/base/dataarray.py#L163). It should be:

```suggestion
        It will have a ``hist_dim_{var_name}`` dimension and a ``plot_axis`` dimension
```